dtests/src/test/scala/io/snappydata/hydra/hiveThriftServer/HiveThriftServer.scala

```diff
@@ -0,0 +1,463 @@
+    pw.println("Case 5 : Create row table from beeline, insert,update,delete from beeline...")
+    executeUseSchema(hiveThriftServer, "app", snc, spark)
+    createRowOrColumnTableFromBeeline(hiveThriftServer,
+      "create TablE if not exists Emp5(id long, name String) using row", pw)
+    insertIntoTableFromBeeline(hiveThriftServer, "insert into Emp5 select " +
+      "id, concat(id, '_Snappy_TIBCO') from range(4000000)", snc, spark,
+      pw, sqlContext, "Emp5", "row")
+    executeShowTables(hiveThriftServer, "show TabLES in app", pw)
+    updateTableFromBeeline(hiveThriftServer,
+      "update Emp5 set id = " +
+        "id+5 where id < 150", snc, spark, pw, sqlContext, "Emp5", "row")
+    deleteFromTableFromBeeline(hiveThriftServer, "delete from Emp5",
+      snc, spark, pw, sqlContext, "Emp5", "row")
+    executeDropTableFromBeeLine(hiveThriftServer, "drop table if exists Emp5")
+
+    pw.println("-------------------------------------------------------------------------------")
+    pw.println("Case 6 : Create column table from beeline, insert,update,delete from Snappy...")
+    createRowOrColumnTableFromBeeline(hiveThriftServer,
+      "create TablE if not exists Emp6(id long, name String) using column", pw)
+    insertIntoTableFromSnappy(hiveThriftServer, "insert into Emp6 select " +
+      "id, concat(id, '_Snappy_TIBCO') from range(50000000)", snc,
+      spark, pw, sqlContext, "Emp6", "column")
+    executeShowTables(hiveThriftServer, "show TabLES in app", pw)
+    updateTableFromSnappy(hiveThriftServer,
+      "update Emp6 set name = " +
+        "'_TIBCO' where id > 1000 and id < 2000", snc, spark, pw, sqlContext, "Emp6", "column")
+    deleteFromTableFromSnappy(hiveThriftServer, "delete from Emp6",
+      snc, spark, pw, sqlContext, "Emp6", "column")
+    executeDropTableFromSnappy(snc, "drop table if exists Emp6")
+
+    pw.println("-------------------------------------------------------------------------------")
+    pw.println("Case 7 : Create row table from Snappy, insert,update,delete from BeeLine...")
+    createRowOrColumnTableFromSnappy(snc,
+      "create TablE if not exists Emp7(id long, name String) using row", pw)
+    insertIntoTableFromBeeline(hiveThriftServer, "insert into Emp7 select " +
+      "id, concat(id, '_Snappy_TIBCO') from range(4000000)", snc,
+      spark, pw, sqlContext, "Emp7", "row")
+    executeShowTables(hiveThriftServer, "show TabLES in app", pw)
+    updateTableFromBeeline(hiveThriftServer,
+      "update Emp7 set id = " +
+        "id+15 where id < 200", snc, spark, pw, sqlContext, "Emp7", "row")
+    deleteFromTableFromBeeline(hiveThriftServer, "delete from Emp7",
+      snc, spark, pw, sqlContext, "Emp7", "row")
+    executeDropTableFromBeeLine(hiveThriftServer, "drop table if exists Emp7")
+
+    pw.println("-------------------------------------------------------------------------------")
+    pw.println("Case 8 : Create column table from Snappy, insert,update,delete from Snappy...")
```

Comment:
We do not need this test case for hive thrift server testing. We can have a test scenario with operations performed from beeline.